cea/datamanagement/format_helper/cea4_verify.py

```
@@ -78,6 +106,95 @@ def path_to_input_file_without_db_4(scenario, item):
+    # Determine file type and load the data
+    if file_path.endswith('.csv'):
+        try:
+            df = pd.read_csv(file_path)
+            col_attr = 'Column'
+        except Exception as e:
+            raise ValueError(f"Failed to read .csv file: {file_path}. Error: {e}")
+    elif file_path.endswith('.shp'):
+        try:
+            gdf = gpd.read_file(file_path)
+            df = pd.DataFrame(gdf.drop(columns='geometry'))  # Drop geometry to validate non-spatial data
+            col_attr = 'Attribute'
```

Ruff (0.8.2)

Local variable `col_attr` is assigned to but never used

Remove assignment to unused variable `col_attr`

(F841)